src/sentry/static/sentry/app/views/performance/trends/utils.tsx

```diff
@@ -331,6 +343,7 @@ export function normalizeTrends(data: Array<ProjectTrend>): Array<NormalizedProj
 export function normalizeTrends(
   data: Array<TrendsTransaction | ProjectTrend>
 ): Array<NormalizedTrendsTransaction | NormalizedProjectTrend> {
+  const received_at = moment(); // Adding the received time for the transaction so calls to get baseline always line up with the transaction
```

Comment:
Does this need to be part of the transformed result, you could make this where it is needed?

Reply:
I could make this elsewhere, but it should be created when you receive the trend. If you make it later (like create a moment it when you call it) you can suffer from skew, and could query baseline that is no longer in the time range for low count transactions. It also should be the same for all of them since you receive the data at the same time.

Reply:
Fair point, I didn't consider that time would slide forward between calls.